Packs/Vectra_AI/Integrations/VectraAIEventCollector/VectraAIEventCollector.yml

```diff
@@ -0,0 +1,66 @@
+category: Network Security
+commonfields:
+  id: VectraAIEventCollector
+  version: -1
+configuration:
+- name: url
+  defaultvalue: https://dev.vectra.ai/
+  display: Your server URL
+  required: true
+  type: 0
+- name: credentials
+  displaypassword: API Token
+  additionalinfo: The API Token to use for authentication.
+  required: true
+  hiddenusername: true
+  type: 9
+- name: first_fetch
+  defaultvalue: 3 days
+  display: First fetch time
+  required: false
+  type: 0
+  section: Collect
+  advanced: true
+- name: fetch_limit
+  defaultvalue: '1000'
+  display: Fetch Limit
+  additionalinfo: Maximum amount of detections to fetch. Audits API does not include a fetch limit therefore this configuration is only relevant to detections.
+  required: false
+  type: 0
+  section: Collect
+  advanced: true
+- name: insecure
+  display: Trust any certificate (not secure)
+  required: false
+  type: 8
+- name: proxy
+  display: Use system proxy settings
+  required: false
+  type: 8
+description: 'Collects Vectra Detections and Audits into XSIAM Events.'
+display: 'Vectra AI Event Collector'
+name: VectraAIEventCollector
+script:
+  commands:
+  - name: vectra-get-events
+    description: Fetches events (detections and audits) from Vectra.
+    arguments:
+    - name: should_push_events
+      auto: PREDEFINED
+      description: If true, the command will create events, otherwise it will only display them. Default is false.
+      isArray: false
+      predefined:
+      - 'true'
+      - 'false'
+      required: true
+  isfetchevents: true
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/py3-native:8.2.0.57778
```

Comment:
the native image shouldnt be used here, we should be using python3 instead.
@GuyAfik should this be added as a validation?

Reply:
I think its a good idea to add a validation that verifies it so we won't change the docker image to native images by mistakes when testing, ill open an issue about it.